docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

```diff
@@ -0,0 +1,118 @@
+# Getting Started with Open Enclave on Windows for systems with support for SGX1
+
+## Platform requirements
+
+Intel® X86-64bit architecture with SGX1
+
+A version of Windows OS with native support for SGX features:
+- For server: Windows Server 2016
+- For client: Windows 10 64-bit version 1709 or newer
+- To check your Windows version, run winver on the command line.
+
+## Install Git and Clone the Open Enclave SDK repo
+
+Download and install Git for Windows from [here](https://git-scm.com/download/win)
+
+Clone the Open Enclave SDK
+
+```powershell
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+## Install project prerequisites
+
+First, change directory into the openenclave repository:
+
+```powershell
+cd openenclave
+```
+
+To deploy all the prerequisities for building Open Enclave, you can run the following from powershell. Note that the Data Center Attestation Primitives Client is not used for attestation on systems which have support for SGX1 without support for Flexible Launch Control (FLC).
+
+```powershell
+cd scripts
+.\install-windows-prereqs.ps1 -InstallPath PATH_TO_OE_REPO -LaunchConfiguration SGX1 -DCAPClientType None
+```
+
+As an example, if you cloned Open Enclave SDK repo into C:\openenclave, you would run the following:
```

Comment:
I think we can make this whole section simpler by assuming the user knows what a Git repo is.